flake8_annotations/__init__.py

```diff
@@ -60,8 +61,9 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         """Format the Argument object into its "official" representation."""
         return (
-            f"Argument({self.argname!r}, {self.lineno}, {self.col_offset}, {self.annotation_type}, "
-            f"{self.has_type_annotation}, {self.has_3107_annotation}, {self.has_type_comment})"
+            f"Argument(argname={self.argname!r}, lineno={self.lineno}, col_offset={self.col_offset}, "  # noqa: E501
+            f"annotation_type={self.annotation_type}, has_type_annotation={self.has_type_annotation}, "  # noqa: E501
+            f"has_3107_annotation={self.has_3107_annotation}, has_type_comment={self.has_type_comment})"  # noqa: E501
```

Comment:
```suggestion
            f"Argument(argname={self.argname!r}, lineno={self.lineno}, col_offset={self.col_offset}, "    # noqa: E501
            f"annotation_type={self.annotation_type}, has_type_annotation={self.has_type_annotation}, "   # noqa: E501
            f"has_3107_annotation={self.has_3107_annotation}, has_type_comment={self.has_type_comment})"  # noqa: E501
```
PEP8 allows this, and I really like it. Maybe you don't, that's fine too.

Reply:
This does look better but black will strip it out so will leave as-is.